From 366dc26347a6d5b16a20112ee7aa8066a4e887e5 Mon Sep 17 00:00:00 2001
From: Alvaro-Karsz <alvaro.karsz@solid-run.com>
Date: Mon, 8 Nov 2021 16:17:53 +0200
Subject: [PATCH] Remove MAC address handle from Device-tree

CPSW MAC address is read from EFUSE, and is defined in Device-tree.

Remove this functionality.

From now on, the MAC Address handle is done in U-boot.
---
 arch/arm64/boot/dts/ti/am642-solidrun.dts | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/ti/am642-solidrun.dts b/arch/arm64/boot/dts/ti/am642-solidrun.dts
index debe5d519..162df0e3e 100644
--- a/arch/arm64/boot/dts/ti/am642-solidrun.dts
+++ b/arch/arm64/boot/dts/ti/am642-solidrun.dts
@@ -602,6 +602,15 @@ cpts@3d000 {
 &cpsw_port1 {
 	phy-mode = "rgmii-rxid";
 	phy-handle = <&cpsw3g_phy0>;
+	
+
+	/*
+	Parameter is defined in DTSI.
+	Overwrite it with an empty entry, The MAC address is read from efuse during U-boot
+	No need to do it again in kernel
+	*/
+
+	ti,syscon-efuse = <>;
 };
 
 &cpsw_port2 {
-- 
2.25.1

